src/libs/ShareManager/ShareType.js

```diff
@@ -0,0 +1,7 @@
+/**
+ * Types of shared content.
+ */
+export default {
+    FILE: 'file',
+    TEXT: 'text',
```

Comment:
Not sure if this was addressed before, but just curious if these are the only two types we need. Is something like an `image` included in `file`?

Reply:
Currently there's only two ways of posting something in your codebase:
`addAction(reportID, text);` for text
`addAction(reportID, '', file)` for any kind of file

See
https://github.com/Expensify/Expensify.cash/blob/83ad99b26de023eac1825f12b65d6928f76b66cc/src/libs/actions/Report.js#L561

So I've added only this two types mainly to differentiate the ways to post. 
Any kind of attachment (image, pdf, zip) are posted the same way, so I didn't see the need to add more types.

Reply:
gotcha! Thanks for the explanation